src/shared/style.scss

```diff
@@ -4,6 +4,11 @@
 // Base
 @import 'style/common';
 @import 'style/layout';
+@import 'style/icons';
+
+// grid
+@import 'style/bootstrap-grid';
```

Comment:
Why do we need bootstrap grid?

Reply:
The `col-*` classes are used in the name value editor component.